browser/ui/webui/brave_wallet/panel_handler/wallet_panel_handler.cc

```diff
@@ -33,6 +45,37 @@ void WalletPanelHandler::CloseUI() {
   }
 }
 
+void WalletPanelHandler::AddEthereumChainApproved(const std::string& payload,
+                                                  int32_t tab_id) {
```

Comment:
We need test coverage for `AddEthereumChainApproved` and `AddEthereumChainCanceled`.